filebeat/filebeat.reference.yml

```diff
@@ -531,6 +531,12 @@ filebeat.inputs:
   # Default is 5s.
   #multiline.timeout: 5s
 
+  # To aggregate constant number of lines into a single event use the count mode of multiline.
+  #multiline.type: count
+
+  # The number of lines to aggregate into a single event.
+  #multiline.lines_count: 3
```

Comment:
In case we introduce this new member I would name it count_lines to be inline with max_lines